### PR TITLE
add RavenService#captureBreadcrumb

### PR DESCRIPTION
**Pull request by marcoow**
This exposes raven's `captureBreadcrumb` method via the `raven` service.

### CODE REVIEW

File: addon/services/raven.js [hunk 1]
```diff
@@ -158,6 +158,21 @@ export default Service.extend({
   },
 
   /**
+   * Tries to have Raven capture breadcrumb, or throw it.
```
Comment (Turbo87) [inline addon/services/raven.js]: `throw it` seems wrong here

File: addon/services/raven.js [hunk 2]
```diff
@@ -158,6 +158,21 @@ export default Service.extend({
...
+   *
+   * @method captureBreadcrumb
+   * @param {Error} error The breadcrumb to capture
+   * @throws {Error} An error if Raven cannot capture the exception
```
Comment (Turbo87) [inline addon/services/raven.js]: `@param` and `@throws` are also wrong

File: addon/services/raven.js (hunk 1 above)
Comment (marcoow) [reply addon/services/raven.js]: 🤦‍♂️ fixed

File: addon/services/raven.js (hunk 2 above)
Comment (marcoow) [reply addon/services/raven.js]: 🤦‍♂️ fixed

**Pull request merged**